extensions/amp-analytics/0.1/events.js

```diff
@@ -447,21 +477,144 @@ export class TimerEventTracker extends EventTracker {
         'Bad maxTimerLength specification');
     const callImmediate = 'immediate' in timerSpec ?
         Boolean(timerSpec['immediate']) : true;
+    const timerStart = 'startSpec' in timerSpec ? timerSpec['startSpec'] : null;
```

Comment:
I am thinking maybe we can do 
```
'tracker': {
  'mytracker1': {
    'on': 'click'
    ...
  }
}
```
then we can have
```
'triggers': {
  'mytriggers': {
     'request': '........',
     'on': 'timer',
     'startSpec': {
        'tracker': 'mytracker1',
      }
   }
}
```
so we don't need to specify the spec multiple times. Not in this PR : )

Reply:
For the case where another tracked event also has a timer, this would be great. But we still need to cover the case where timer events are not necessarily tracked individually, since otherwise they're just duplicate info on the ping.